src/web/app/src/components/SignUp/Forms/Review.tsx

```diff
@@ -78,6 +78,7 @@ const useStyles = makeStyles((theme: Theme) =>
       textAlign: 'start',
     },
     blogRss: {
+      maxWidth: '300px',
```

Comment:
Question: where did `300px` come from?

Reply:
The UI shifts around if I use % instead, so I use px. I judged it by the way the UI looks and picked a number that seems like a good place to stop the overflow.

Reply:
I'd feel better if you looked at the surrounding code to figure out what the width used elsewhere in parent components is.  If you have to use a magic number like this, so be it.  But if you can avoid it, that would be better.